dto/data_model_dto.go

```diff
@@ -106,6 +107,7 @@ func adaptDataModelField(field models.Field) Field {
 
 func adaptDataModelLink(linkToSingle models.LinkToSingle) LinkToSingle {
 	return LinkToSingle{
+		Id:                     linkToSingle.Id,
```

Comment:
The main thing: I need link ids for pivot creation

Reply:
my bad, I thought I added this